### DISCUSSION TITLE
What happens after yarn run watch?

### DISCUSSION THREAD

**Discussion by AntoineOrla** [mxstbr/karabiner#7] [Q&A]
Hi I'm very new to the world of terminal and github but I am very intrigued by your karabiner setup and I'd love to be able to use it. I think I've done all the steps right and the last prompt I got from the terminal after typing yarn run watch was this:

yarn run v1.22.19
$ nodemon --exec yarn build -e ts
[nodemon] 2.0.20
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: ts
[nodemon] starting `yarn build`
$ tsm rules.ts
(node:6093) ExperimentalWarning: Custom ESM Loaders is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6093) DeprecationWarning: Obsolete loader hook(s) supplied and will be ignored: getFormat, transformSource
[nodemon] clean exit - waiting for changes before restart

Is this what it was supposed to say and whats next? 

Best regards,
Antoine

**Comment by mxstbr** [comment]
Yep, that's exactly right! `yarn run watch` watches the `rules.ts` for changes and will rebuild the `karabiner.json` output file in response.
